grafana/dashboards/brigade.json

```diff
@@ -6,7 +6,7 @@
   "version": 4,
   "refresh": "5s",
   "style": "dark",
-  "editable": false,
+  "editable": true,
```

Comment:
I don't think you meant to commit this change.

Reply:
I did not, I'm trying to revert the change but I think I messed up the git surgery

Reply:
It should be fixed now @krancour